vortex-array/src/array/chunked/mod.rs

```diff
@@ -68,15 +68,6 @@ impl ChunkedArray {
             .last()
             .vortex_expect("Chunk ends is guaranteed to have at least one element");
 
-        let stats = chunks
-            .iter()
-            .map(|chunk| chunk.statistics().to_set())
-            .reduce(|mut acc, stats| {
-                acc.merge_ordered(&stats);
-                acc
-            })
-            .unwrap_or_default();
```

Comment:
I think given this only merges present stats and each chunk will already cache their stats this seems wasteful if no one asked for stats